maps/sccv_horizon/code/sccv_horizon_overmap.dm

```diff
@@ -43,27 +43,31 @@
 
 /obj/effect/overmap/visitable/ship/landable/intrepid
 	name = "Intrepid"
+	class = "SCCV"
+	designation = "Intrepid"
 	desc = "A standard-sized unarmed exploration shuttle manufactured by Hephaestus, the Pathfinder-class is commonly used by the corporations of the SCC. Featuring well-rounded facilities and equipment, the Pathfinder is excellent, albeit pricey, platform. This one's transponder identifies it as the SCCV Intrepid."
```

Comment:
It's not unarmed anymore, is it? should that be changed?

Reply:
true